docs/get-started/start-teku.md

```diff
@@ -125,6 +125,36 @@ Content-Length: 0
   </TabItem>
 </Tabs>
 
+## Exit Codes
+
+**Exit Code 1**
+
+This code indicates a scenario where Teku has exited with a fatal error, however restarting Teku without changes is a logical step in correcting it. This code represents the error is related to something external to Teku.
```

Comment:
This code ... fatal error, however restarting ... = This code ... fatal error; however, restarting ...